bad/source/lex.rs

```diff
@@ -0,0 +1,87 @@
+use logos::Logos;
+
+#[derive(Logos, Debug)]
+pub enum Token {
+	#[regex(r"[ \t\n\f]")]
+	Whitespace,
+	
+	#[regex(r"[a-zA-Z_]+[a-zA-Z0-9_]*")]
+	Identifier,
+	Symbol,

```

Comment:
What's the difference between these guys?

Reply:
It was meant to cover random punctuators that don't exist as identifiers, but that's honestly overkill for a B parser!